cli/dcoscli/package/main.py

```diff
@@ -366,9 +365,6 @@ def _install(package_name, package_version, options_path, app_id, cli, app,
 
     if app and pkg.has_mustache_definition():
 
-        # render options before start installation
-        options = pkg.options(user_options)
```

Comment:
Since `/package/install` renders the options, this didn't seem necessary. It did require one of the tests to be modified, but the change seems harmless.